server/src/test/java/io/spine/examples/shareaware/server/e2e/given/EntitySubscription.java

```diff
@@ -68,10 +97,29 @@ private void setState(S value) {
 
         private S state() {
             try {
-                return future.get();
+                S value = future.get(10, SECONDS);
+                return value;
+            } catch (InterruptedException | ExecutionException | TimeoutException e) {
+                throw illegalStateWithCauseOf(e);
+            }
+        }
+
+        private S onceUpdated() {
```

Comment:
Let's document this method as well.

I think it is now called `waitForUpdate()`.